types/guacamole-client/lib/Client.d.ts

```diff
@@ -221,10 +221,10 @@ export class Client {
      * Sends a key event having the given properties as if the user
      * pressed or released a key.
      *
-     * @param pressed Whether the key is pressed (true) or released (false).
+     * @param pressed Whether the key is pressed (1) or released (0).
      * @param keysym The keysym of the key being pressed or released.
      */
-    sendKeyEvent(pressed: boolean, keysym: number): void;
+    sendKeyEvent(pressed: 0 | 1, keysym: number): void;
```

Comment:
Guacamole's jsdoc lies about this being a boolean. Both in their client [implementation](https://github.com/apache/guacamole-client/blob/11c1575daf58111049f4c5cc1cd6ea1b01d5ee9c/guacamole/src/main/frontend/src/app/client/directives/guacClient.js#L425) and rhyme.com passing a boolean doesn't work; one has to pass `1` or `0`